app_dart/test/request_handlers/github/webhook_subscription_test.dart

```diff
@@ -2616,11 +2619,44 @@ void foo() {
         merged: true,
       );
 
+      when(gerritService.findMirroredCommit(any, any)).thenAnswer(
```

Comment:
Instead of using mocks, can we use a fake?

Reply:
Swapped it out 👍